.vscode/launch.json.default

```diff
@@ -0,0 +1,49 @@
+{
+    // Sample launch.json for debugging CLI commands in VS Code.
+    // Copy to launch.json as needed.
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Validate",
+            "type": "python",
+            "request": "launch",
+            "pythonPath": "${config:python.pythonPath}",
+            "program": "${workspaceFolder}/c7n/cli.py",
+            "cwd": "${workspaceFolder}/c7n",
+            "console": "integratedTerminal",
+            "env": { "PYTHONPATH": "${workspaceFolder}"},
+            "envFile": "${workspaceFolder}/.env",
+            "debugOptions": [],
+            "internalConsoleOptions": "neverOpen",
+            "args": [
+                "schema",
+                "aws.ec2"
+            ],
+            "exceptionHandling":{
+                "ignore": ["copy.Error"]
+            }
+        },
+        {
+            "name": "Run",
+            "type": "python",
+            "request": "launch",
+            "pythonPath": "${config:python.pythonPath}",
+            "program": "${workspaceFolder}/c7n/cli.py",
+            "cwd": "${workspaceFolder}",
+            "console": "integratedTerminal",
+            "env": { "PYTHONPATH": "${workspaceFolder}"},
+            "envFile": "${workspaceFolder}/.env",
+            "debugOptions": [],
+            "internalConsoleOptions": "neverOpen",
+            "args": [
+                "run",
+                "-s",
+                "output",
+                "test.yml"
```

Comment:
we would need to add one of these to the repo else its user specific config, a little unclear if this was an incidental add or intentional.

Reply:
I had just made up some parameters to include in that launch.json.default as a sample so people would know how to use it if they chose to copy it into their launch.json.  Maybe launch.json.sample would be a better filename.  (Or I can just leave it out :) )

Reply:
Ok so I added an ignore to everything in `.vscode/` except `.vscode/*.sample` and added launch and settings samples (and removed settings.json).  That should be very visible/obvious for anyone in there trying to edit/figure out their workspace or debug settings without making it harder for them to edit their own copies.